### DISCUSSION TITLE
How to use HD Action to write the javascript action?

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/helpdesk#1749] [Q&A] [answered]
I am trying to add action to open the document, by doing this.

![image](https://github.com/frappe/helpdesk/assets/1973598/f368fd47-063b-43db-9db3-c77ddfd6f413)

But `doc.name` throw error. How to properly get the document value?

Thanks

**Comment by ssiyad** [comment, marked as answer]
@kittiu use `this.name` instead of `doc.name`. `this` is same as `doc` here
